custom_components/powercalc/data/lidl/14156506L/model.json

```
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "TAPO P110",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.10.0:docker"
+    },
+    "name": "Livarno Lux smart LED mood light",
+    "standby_power": 0
```

```suggestion
    "standby_power": 0.3
```

Standby power can never be 0. Sometimes smartplugs are unable to measure power values < 0.5W. 
0.3W would be a good estimate of zigbee based bulbs.